taniumrest_connector.py

```diff
@@ -386,7 +404,8 @@ def _handle_list_questions(self, param):
         # Add the response into the data section
         for question in response.get("data", []):
 
-            if question.get("id") and not param.get('list_saved_questions', False) and question.get('query_text') not in question_list:
+            if question.get("id") and not param.get('list_saved_questions', False) \
+                    and question.get('query_text') not in question_list:
```

Comment:
suggestion: What do you think about using brackets to work around the new line instead?
```python
if (question.get("id") and not param.get('list_saved_questions', False)
            and question.get('query_text') not in question_list):
         pass
```

Reply:
Removed the line break. Because, earlier the max line length was 130, but now it is increased to 145. So, no need for a line break.